### PR TITLE
Support for status messages

### PR DESCRIPTION
**Pull request by malditogeek**
Added support for status messages. Didn't modify the `send` function to not break backwards compatibility.

/cc @trevorah

### CODE REVIEW

File: lib/rooms.js [hunk 1]
```diff
@@ -53,6 +53,12 @@ Room.prototype.send = function(message, cb) {
   return cb ? msg.nodeify(cb) : msg;
 };
 
+Room.prototype.sendStatus = function(message, cb) {
+  var msg = this.client.post(this.path + '/' + this.id + '/chatMessages', {body: {text: message, status: true}});
+  return cb ? msg.nodeify(cb) : msg;
```
Comment (suprememoocow) [inline lib/rooms.js]: No need to do this @malditogeek.. Q will do it for you...
```
return this.client.post(this.path + '/' + this.id + '/chatMessages', { body: { text: message, status: true }})
           .nodeify(cb)

```

File: lib/rooms.js [hunk 2]
```diff
@@ -53,6 +53,12 @@ Room.prototype.send = function(message, cb) {
...
+};
+
+
```
Comment (suprememoocow) [inline lib/rooms.js]: Such newline!

File: lib/rooms.js (hunk 1 above)
Comment (malditogeek) [inline lib/rooms.js]: oh, so if the callback is undefined it'll return the promise? 

will change this in another PR

Comment (suprememoocow) [inline lib/rooms.js]: Thats correct. Saves you having to do conditionals for handling promises and callbacks in the same code

**Pull request merged**